docs/guides/mysql/clustering/group-replication/group_replication_single_primary.md

```diff
@@ -0,0 +1,495 @@
+---
```

Comment:
KubeDB current website does not support three level folder. for now, keep group replication on clustering folder.